src/main/scala/org/hammerlab/guacamole/DistributedUtil.scala

```diff
@@ -312,19 +321,24 @@ object DistributedUtil extends Logging {
     skipEmpty: Boolean,
     function: (Pileup, Pileup) => Iterator[T],
     reference: ReferenceGenome): RDD[T] = {
+
+    val referenceBC = reads1.context.broadcast(reference)
```

Comment:
I added overloaded versions of the pileupFlatmap functions that take a `Broadcast[ReferenceGenome]` directly.